bellows/zigbee/zcl/foundation.py

```diff
@@ -101,7 +101,6 @@ class Discrete:
 DATA_TYPE_IDX = {
     t: tidx
     for tidx, (tname, t, ad) in DATA_TYPES.items()
-    if ad is Analog
```

Comment:
Yes you are right, I have just tested with a t.uint16_t type and I get INVALID_DATA_TYPE back.

So I also think that the only correct way is option 1. 
Because option 2 will not work, because we will still hit the many-to-one issue.
And options 3 will have the wrong data type, if it is not the first one in the list.

So how should we solve this?